reaper/reaper.go

```diff
@@ -79,45 +79,59 @@ func (reaper reaper) getPods() *v1.PodList {
 	return podList
 }
 
-func (reaper reaper) reapPod(pod v1.Pod, reasons []string) {
+func (reaper reaper) reapPod(pod v1.Pod, reasons []string, reapedPods int) {
 	deleteOptions := &metav1.DeleteOptions{
 		GracePeriodSeconds: reaper.options.gracePeriod,
 	}
+
+	podLog := logrus.WithFields(logrus.Fields{
+		"pod":     pod.Name,
+		"reasons": reasons,
+	})
+
 	if reaper.options.dryRun {
-		logrus.WithFields(logrus.Fields{
-			"pod":     pod.Name,
-			"reasons": reasons,
-		}).Info("pod would be reaped but pod-reaper is in dry-run mode")
+		podLog.Info("pod would be reaped but pod-reaper is in dry-run mode")
```

Comment:
I'm not sure how I feel about separating the pod/reason from the dry-run message here, but I'm not using the structured logging for anything. It seems reasonable to me.

Reply:
I'm fine with reverting those log refactors, if that's what you mean?

Reply:
My preference would be to have all the logging messages include at least the "field" `"pod"` just in case someone is using a log parser to search.

I want to avoid a case where someone is searching on the "pod" field for their specific pod and missing something like "dry-run". 

It's not a use case I have but I do remember someone asking for more "structured logging" (with Jason formatting) so they could do a search like that.

I'm hoping my explanation makes sense because it's going a long ways back in memory!

Reply:
perhaps we're misunderstanding each other here, the changes to the logging I did here don't change the _output_ of the logs but merely the fact that we need to pass the same fields to logrus on each invocation. So by preparing `podLog` once and calling that instead of `logrus` directly, it will have the `"pod"` and `"reasons"` field each time. Does that make sense to you?

Reply:
I see it now! Pretty sure you're absolutely right and I just missed it!
My bad.